docker/php/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM php:apache
+
+# Settings
+WORKDIR /var/www/html
+ENV COMPOSER_ALLOW_SUPERUSER=1
+
+# Install server dependencies
+RUN apt-get update &&\
+    apt-get install -y git unzip vim zlib1g-dev libzip-dev libpng-dev
+
+# Install PHP extensions
+RUN docker-php-ext-install mysqli pdo pdo_mysql zip gd
+
+# Enable PHP modules
+RUN a2enmod rewrite
+
+# Install composer
+RUN curl -sS https://getcomposer.org/installer -o composer-setup.php &&\
+    php composer-setup.php --install-dir=/usr/local/bin --filename=composer
+
+# Install OctoberCMS
+RUN git init . &&\
+    git remote add origin https://github.com/octobercms/october.git &&\
+    git pull origin master &&\
```

Comment:
I might look into making this configurable, so that we can test (and deploy) stable releases instead of the `master` branch. This would also make it possible to test the Docker setup against the current CMS and plugins versions that we have deployed on godotengine.org (OctoberCMS 419).